graphics/ogre/Portfile

```diff
@@ -21,83 +21,65 @@ long_description    OGRE (Object-Oriented Graphics Rendering Engine) is a \
                     library abstracts all the details of using the \
                     underlying system libraries like Direct3D and \
                     OpenGL and provides an interface based on world \
-                    objects and other intuitive classes. \
-                    NOTE: \
-                    This version is so far the only version which produces \
-                    normal non-bundled executables that can run on any \
-                    64-bit MacOS X machine without depending on MacPorts or \
-                    any other libraries. This version is also the only \
-                    version that allows building fully functioning CMake \
-                    OGRE projects on MacOS X without using Xcode.
+                    objects and other intuitive classes.
 
 homepage            https://www.ogre3d.org/
 platforms           darwin
 supported_archs     x86_64
-master_sites        sourceforge:project/ogre/ogre/${branch}
-checksums           rmd160  fea467f35aaf0fd5926573de4f0348ed44191893 \
-                    sha1    41acccfbbf7297c91cda78a0ce8a053e56505f04
-distname            ogre_src_v[strsed ${version} {g/\./-/}]
-use_bzip2           yes
 
-boost.version       1.71
+checksums           rmd160  52c0dd24b8ec427d35a97c2b73746298653ce8db \
+                    sha1    272345936aa286f848bb8aa1691521b34044ba53 \
+                    sha256  8d31a79d37f74e5af25e56a0a70928079e771f7cea50ef041e04bf235c3f701a \
+                    size    91796574
 
-depends_build-append port:doxygen path:bin/dot:graphviz port:pkgconfig
+use_xcode           yes
 
-depends_lib-append  port:libzzip port:zlib port:bzip2 port:freeimage \
-                    port:freetype port:ois
+cmake.build_type    Release
```

Comment:
ideally you should not set the `build_type`. It will be set automatically to `MacPorts`, which will honor all the typical flags set by MacPorts

Reply:
@reneeotten yes, but this software is required to used [specified build type](https://ogrecave.github.io/ogre/api/latest/building-ogre.html#autotoc_md263). It catches when build it type isn't provided and used [one of acceptable value](https://github.com/OGRECave/ogre/blob/master/CMakeLists.txt#L95).

Default `MacPorts` brokes too many things. The right way is implement support of `MacPorts` build type inside, but it requires huge amount of work with not clear benefits. Why? Because this port was broken for years and I just fixed it with smaller possible impact.

Reply:
no completely, the statement you link points to just says that it cannot be empty. It builds fine for me without this line, so can you please try and remove `cmake.build_type    Release` as that will set `CMAKE_BUILD_TYPE` to `MacPorts and is thus doesn't reach that check.

Reply:
@reneeotten let me try, maybe it isn't nessaray after all.

Reply:
@reneeotten without `cmake.build_type Release` it builds different way and resulted port very different. Just a beginning of diff for `port content ogre`:
```
--- macports-build-type.txt     2021-09-22 16:37:43.000000000 +0200
+++ release-build-type.txt      2021-09-22 16:42:55.000000000 +0200
@@ -1,4 +1,8 @@
 Port ogre contains:
+  /opt/local/bin/OgreAssimpConverter
+  /opt/local/bin/OgreMeshUpgrader
+  /opt/local/bin/OgreXMLConverter
+  /opt/local/bin/VRMLConverter
   /opt/local/include/OGRE/Bites/OgreAdvancedRenderControls.h
   /opt/local/include/OGRE/Bites/OgreApplicationContext.h
   /opt/local/include/OGRE/Bites/OgreApplicationContextBase.h
@@ -538,6 +542,621 @@
   /opt/local/include/OGRE/Volume/OgreVolumeSimplexNoise.h
   /opt/local/include/OGRE/Volume/OgreVolumeSource.h
   /opt/local/include/OGRE/Volume/OgreVolumeTextureSource.h
+  /opt/local/lib/Codec_Assimp.framework/Codec_Assimp
+  /opt/local/lib/Codec_Assimp.framework/Headers
+  /opt/local/lib/Codec_Assimp.framework/Resources
```